### PR TITLE
Added welcome page and fixes minor tutorial issues

### LINKED ISSUE #151: Welcome/Overview

### LINKED ISSUE #339: Drawbuffers Deceleration incorrect
In one of the tutorials there is a draw buffers, not a render targets.

### PR DESCRIPTION
**Pull request by WhyFenceCode**
feat: added a welcome page to the docs, as well as helpful navigation. #151 **ninjamike Review**
fix: fixed incorrect deceleration of rendertargets as drawbuffer. #339  **jbritain Review**
Other minor spelling fixes and minor changes to formatting of the home and welcome pages

### CODE REVIEW

File: src/content/docs/reference/overview.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Overview
+title: Welcome
```
Comment (jbritain) [inline src/content/docs/reference/overview.mdx:2]:
```suggestion
title: Welcome...
```

File: src/content/docs/reference/overview.mdx [hunk 2]
```diff
@@ -9,6 +9,14 @@ sidebar:
 
 import { LinkCard, CardGrid } from '@astrojs/starlight/components';
 
+To the Iris docs!
```
Comment (jbritain) [inline src/content/docs/reference/overview.mdx:12]:
```suggestion
...to the Iris docs!
```

File: src/content/docs/reference/overview.mdx [hunk 3]
```diff
@@ -9,6 +9,14 @@ sidebar:
...
+
+Iris is a shader loader that is integrated with sodium, with support for modern shaders, built from the ground up for ease of use. Weather your new to shader development, want to migrate your shader from optifine, or just need a refresher on what Iris can do, you have come to the right place.
```
Comment (jbritain) [inline src/content/docs/reference/overview.mdx:14]:
```suggestion
Iris is a shader loader that is integrated with [Sodium](https://modrinth.com/mod/sodium/), with support for modern shaders, built from the ground up for ease of use. Whether you're new to shader development, want to migrate your shader from Optifine, or just need a refresher on what Iris can do, you can find everything you need to know right here.
```

File: src/content/docs/reference/overview.mdx [hunk 4]
```diff
@@ -9,6 +9,14 @@ sidebar:
 
 import { LinkCard, CardGrid } from '@astrojs/starlight/components';
 
+To the Iris docs!
+
+Iris is a shader loader that is integrated with sodium, with support for modern shaders, built from the ground up for ease of use. Weather your new to shader development, want to migrate your shader from optifine, or just need a refresher on what Iris can do, you have come to the right place. 
+
+If you can't find what you are looking for, reach out to us on the [Iris Discord](https://discord.com/invite/jQJnav2jPu).
+
+Now, if you are feeling a bit lost, why dont you look into the items bellow.
```
Comment (jbritain) [inline src/content/docs/reference/overview.mdx:18]:
```suggestion
If you can't find what you are looking for, reach out to us on the [Iris Discord](https://discord.com/invite/jQJnav2jPu).

If you're feeling a bit lost, why don't you have a look at the items below.
```

**Review by jbritain — COMMENTED** [review]
I also think we need to rethink the "quick reference", especially now we have more changes. We should probably remove the DH support page until we actually...y'know, write it, and the quick reference can link to stuff like uniforms, constants, and vertex attributes. The message about the sidebar/navbar feels a bit out of place now as well.

File: src/content/docs/reference/overview.mdx (hunk 3 above)
Comment (jbritain) [reply src/content/docs/reference/overview.mdx:14]: Also I feel like "modern shaders" is vague and doesn't mean much.

Comment (WhyFenceCode) [reply src/content/docs/reference/overview.mdx:14]: Sounds good tho

**Comment by WhyFenceCode** [comment]
Thank you for all your feedback. I will work on rewriting the DH shaders thing soon.

**Comment by WhyFenceCode** [comment]
Changed DH link to the uniforms overview (which makes more sense)

**Pull request merged**